### ISSUE TITLE
The right-click menu does not display functionality

### ISSUE THREAD

**Issue by pwd9527** [vrtadmin/FIRST-plugin-ida#13]
Environment: win7 x64, ida7.0,  Python 2.7.13
The right-click menu does not display functionality

**Comment by demonduck** [comment]
@pwd9527 can you provide me with more details? What type of binary are you opening up? Have you configured the FIRST plugin to connect to a FIRST server? Please update to the current dev branch (11eca30  ) version of the plugin if you are using the master version. 

Without more information it can be hard to understand why this is happening. Something you could try:
Once in IDA Pro, type the below command into the Output Window (ensure Python is selected, not IDC). Hopefully you will see the same below output, with `MEM_ADDRESS` being unique for the plugin and the server.
```
Python>FIRST.plugin_enabled
True
Python>FIRST.plugin
<__main__.FIRST_FormClass object at MEM_ADDRESS>
Python>FIRST.server
<__main__.Server object at MEM_ADDRESS>
```

**Comment by pwd9527** [comment]
That has been solved,  "11eca30" is ok, Thank you very much.
